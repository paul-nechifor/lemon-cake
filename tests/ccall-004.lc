(ccall
  (dynsym 'libc.so' 'printf')
  'asdf %s %s %s %s %s'
  'a' 'b' 'c' 'd' 'e'
)
