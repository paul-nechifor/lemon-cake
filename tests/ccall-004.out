asdf a b c d e
